### DISCUSSION TITLE
跪求大神们 适配下11哎。感谢感谢！

### DISCUSSION THREAD

**Discussion by cn-king** [MustangYM/WeChatExtension-ForMac#979] [General]
跪求大神们 适配下11哎。感谢感谢！

**Comment by flappybriefs** [comment]
其实11好像能正常使用

**Reply by cn-king** [reply]
您好，谢谢解答！我安装完之后， 打开微信 提示的是 微信遇到问题无法打开。就是下面这个图。请问是我哪一步错啦吗？谢谢啦

**Reply by cn-king** [reply]
![Screen Shot 2021-05-18 at 15 10 31](https://user-images.githubusercontent.com/55311966/118607820-992d1c00-b7eb-11eb-9792-5858360e738b.png)

**Reply by cn-king** [reply]
我的系统版本是11.4，微信版本3.0.3

**Reply by flappybriefs** [reply]
估计是应用签名的问题，重新给微信签名就行了

**Reply by cn-king** [reply]
> 估计是应用签名的问题，重新给微信签名就行了

兄弟这回答有点早专业哇，，能详解下如何操作吗？ 我有些惶恐

**Reply by flappybriefs** [reply]
其实我也是小白😂
你到MacWk上面下载一个macOS小助手，里面可以一键签名或者绕过签名，用这俩方法就能解决

**Reply by cn-king** [reply]
好滴好， 感谢呀！

**Reply by cn-king** [reply]
codesign -f -s - --deep /Applications/WeChat.app

**Reply by flappybriefs** [reply]
不用谢😁 这个小工具挺有用的，我经常用它修复软件
